package.json

```diff
@@ -19,17 +19,17 @@
     "dev-clean": "shx rm -rf packages/**/node_modules && npm uninstall -g langium-workspaces langium-cli generator-langium langium && npm unlink langium-workspaces langium-cli generator-langium langium"
   },
   "devDependencies": {
-    "@types/jest": "^29.0.3",
-    "@types/jest-expect-message": "^1.0.4",
```

Comment:
@kaisalmen I'll defer to @msujew for the [specifics](https://github.com/langium/langium/pull/690#pullrequestreview-1115186651) from PR #690 , but switching to `vitest/globals` results in a linting supernova.

Philosophically, I'm _ok_ with discrete imports on a per-file basis since that's more explicit, but that's a decision I'll defer to the maintainers. I toyed with the idea originally, but the Langium team implemented some fairly elegant test harnesses that I didn't fully understand at the time so I gave up. Theoretically, the jest-style `vitest/globals` would preserve the current test workflow but resolving the typing/linting mess is beyond my skill.